### PR TITLE
Refactored SecurityController

### PR DESCRIPTION
**Pull request by chives**
This PR solves #27

### CODE REVIEW

File: Controller/SecurityController.php [hunk 1]
```diff
@@ -13,55 +13,43 @@
 use Symfony\Component\Form\Extension\Csrf\CsrfProvider\CsrfProviderInterface;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\Security\Core\SecurityContext;
```
Comment (szymach) [inline Controller/SecurityController.php]: it is now unused and should be removed

**Comment by chives** [comment]
@fsi-open

**Comment by szymach** [comment]
:+1:

**Comment by rn0** [comment]
:+1:

**Comment by JarekW** [comment]
:+1:

**Pull request merged**